### PR TITLE
Add list Structure

### PR DESCRIPTION
**Pull request by HAMMAS-SALEEM**
### In this branch, I have worked on these things 📝

✔️ Set up a new project with Webpack.
✔️ Create a markup file and hardcode the basic UI 
✔️ The markup uses the HTML Webpack Plugin.
✔️ Create a javascript entry point file
✔️ Set an array of some simple to-do tasks (array of objects).
✔️ Each task object contains three keys:
   - description [string].
   - completed [bool].
   - index: [number].

✔️ Write a function to iterate over the tasks array and populate an HTML list item element for each task.
✔️ On page load render the dynamically created list of tasks in the dedicated placeholder. 
✔️ The list appears in order of the index values for each task.
✔️ Create a stylesheet and set rules for the To-Do List. 
✔️ CSS is loaded by Webpack Style/CSS Loader.
✔️ Followed GitHub Flow
✔️ Followed Linters
✔️ Followed HTML, CSS, and JavaScript best practices
✔️ Documented work in a professional way

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+- JavaScript
+
+
+## Clone and review it
+
+Run the following commands to clone and run it.
+
+### To clone the repository
+
+  `git@github.com:HAMMAS-SALEEM/ToDoList.git`
```
Comment (HENRYKC24) [inline README.md]: - [ ] [Required] I suggest you specify all the commands for cloning your repository here. In that way, every developer can easily clone the repository without having to try to remember the exact code for that. What about we say: `git clone git@github.com:HAMMAS-SALEEM/ToDoList.git` instead? What do you think?

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+
+### To enter Directory
+
+`cd ToDoList`
+
+### To Install Packages
+
+`npm install`
```
Comment (HENRYKC24) [inline README.md:31]: - [ ] [Required] I also think that after running `npm install`, the tester has to run `npm start` to start the local server. This makes it very easy for whoever is trying to use your application. Let's add that after this line I suggest.

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+
+### To run
```
Comment (HENRYKC24) [inline README.md]: - [ ] [Required] I think you meant `To open with vs-code` here. `code .` opens the project on visual studio code application I guess. What's your thought on this, please?

**Review by HENRYKC24 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUESTED :recycle: 
Changes Requested
## Hi @HAMMAS-SALEEM :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
Hi @HAMMAS-SALEEM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**